js/main.js

```diff
@@ -0,0 +1,137 @@
+const teamEl = document.querySelector('.team');
+const servicesEl = document.querySelector('.services');
+const viewMoreEl = document.querySelector('.view-more');
+const btnMoreEl = document.querySelector('.btn-more');
+
+const teamInfo = [
+  {
+    name: 'Sarah Sinclair',
+    position: 'Android Developer',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam cupiditate ex esse
+    sapiente. Distinctio quae adipisci ipsum sed nihil iste tenetur, minus laboriosam,
+    quos animi eligendi eos, dignissimos perferendis nesciunt?`,
+    imageUrl: ['../imgs/1.png', 'Photo of Sarah'],
```

Comment:
- [ ] All our images for the team section are not being rendered on the page 👇🏼:

  ![Screenshot 2022-06-22 151655](https://user-images.githubusercontent.com/45002654/175029138-287235b4-2689-470e-8332-5c6a62283a22.png)

  This is because of the path definition. Let us realize that the `index.html` and the `imgs` directory are in the same directory. Therefore `./` should be the correct path description and not `../`.

  Please refactor all the paths for images in this section likes so 👇🏼:
  
  ```suggestion
    imageUrl: ['./imgs/1.png', 'Photo of Sarah'],
```

Reply:
Noted. I will work on it.